src/pages/guide/css/preprocess.md

```diff
@@ -263,6 +263,49 @@ In the processed file, this results in the following:
  // Theme widgets
 ```
 
+## Trigger flag deploy styles for only enabled modules
+
+Default all css by default will build to final css no matter modules enabled or disabled. With this flag explain below allow developers able to control how we want css generate
```

Comment:
```suggestion
<InlineAlert variant="info" slots="text"/>

This feature is available in 2.4.8-beta only.

By default, the core code base imports all CSS from all modules, regardless of whether modules are enabled or disabled. This can lead to unnecessarily large CSS files, which can delay browser parsing, especially for stores with many modules and custom styles. To reduce the size of the CSS output and improve browser performance, you can use the `static_content_only_enabled_modules` flag.
```